analyzer/src/funTest/kotlin/AnalyzerFunTest.kt

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2025 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.analyzer
+
+import io.kotest.core.spec.style.WordSpec
+import io.kotest.matchers.collections.containExactly
+import io.kotest.matchers.should
+
+import java.io.File
+import java.io.IOException
+
+import org.ossreviewtoolkit.analyzer.Analyzer.ManagedFileInfo
+import org.ossreviewtoolkit.model.ProjectAnalyzerResult
+import org.ossreviewtoolkit.model.config.AnalyzerConfiguration
+import org.ossreviewtoolkit.model.config.RepositoryConfiguration
+
+class AnalyzerFunTest : WordSpec({
+    "afterResolution()" should {
```

Comment:
Normally, we name this after the function which is tested. 
Also,  the test name does not really match the assertion.

Proposal:  
```
"analyze()" should {
     "make all resolution hooks even if resolveDependencies() throws." {
```

Reply:
Done slightly differently.